sources/engine/Stride.VirtualReality/OpenVR/OpenVR.cs

```diff
@@ -24,8 +24,11 @@ internal static class OpenVR
         /// <remarks>Take care to ensure that the struct has been initialized appropriately, otherwise the struct's fields could contain uninitialized data from the stack.</remarks>
         private static ref T SkipInit<T>(out T value)
         {
-            Unsafe.SkipInit(out value);
-            return ref value;
+            unsafe
```

Comment:
I think it'd be nicer to put unsafe on the method declaration

Reply:
Fixed. I agree